settings/custom-domain.mdx

```diff
@@ -13,7 +13,7 @@ To set up your documentation on a custom subdomain, you'll need to set your desi
 
 ## Dashboard Settings
 
-1. Head over to your [dashboard project](https://dashboard.mintlify.com)
+1. Head over to your [dashboard project](https://dashboard.mintlify.com/settings)
```

Comment:
Should the title of the link also be changed?
```suggestion
1. Head over to your [dashboard project](https://dashboard.mintlify.com)
2. Navigate to the [settings](https://dashboard.mintlify.com/settings) page.
```

Reply:
spot on! thanks!